lib/experimental/fonts/font-library/class-wp-font-family-utils.php

```diff
@@ -107,4 +107,64 @@ function ( $family ) {
 
 		return $font_family;
 	}
+
+	/**
+	 * Generates a slug from font face properties.
+	 *
+	 * Used for comparison with other font faces in the same family.
+	 *
```

Comment:
Maybe we could link to the CSS spec in the comments, so it's easy for folks to find if they want to see the spec details, etc.

Reply:
Great idea! I've added a `@link` in the function's docblock.